tests/php/src/OptionsRESTControllerTest.php

```diff
@@ -93,8 +93,8 @@ public function test_get_items() {
 		$this->assertEquals( $data['preview_permalink'], null );
 		$this->assertEquals( $data['suppressed_plugins'], [] );
 		$this->assertEquals( $data['suppressible_plugins'], [] );
-		$this->assertContains( 'post', wp_list_pluck( $data['supportable_post_types'], 'name' ) );
-		$this->assertEquals( $data['supported_post_types'], [ 'post' ] );
+		$this->assertEquals( [ 'post', 'page', 'attachment' ], wp_list_pluck( $data['supportable_post_types'], 'name' ) );
+		$this->assertEquals( $data['supported_post_types'], [ 'post', 'page' ] );
```

Comment:
Just noticed most of these assertions in this method should be the other way around; `actual` should come after `expected`.

```suggestion
		$this->assertEquals( [ 'post', 'page' ], $data['supported_post_types'] );
```

Reply:
Fixed the other assertion arguments in https://github.com/ampproject/amp-wp/pull/5053/commits/5d6714a514204399ce689be46dce5957c8d21543.